clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -313,23 +313,23 @@ void loop_attrs_compatibility() {
   [[intel::disable_loop_pipelining]]
   [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  // expected-error@+1 {{disable_loop_pipelining and max_interleaving attributes are not compatible}}
+  // expected-error@+2 {{'disable_loop_pipelining' and 'max_interleaving' attributes are not compatible}}
   [[intel::disable_loop_pipelining]]
   [[intel::max_interleaving(0)]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
-  // expected-error@+1 {{disable_loop_pipelining and speculated_iterations attributes are not compatible}}
+  // expected-error@+1 {{'disable_loop_pipelining' and 'speculated_iterations' attributes are not compatible}}
```

Comment:
Shouldn't this be +2 with attributes reversed in diagnostic?

Reply:
Yes, it really should. I had noticed that when making the change but hadn't solved the underlying issue (the way we check for mutual exclusion is causing this issue). I figured this was no worse than the original behavior, but I think I may have a solution for the problem, so let me give that a whirl.

Reply:
This should now be resolved.